mta.yaml

```diff
@@ -62,32 +48,36 @@ modules:
         url: ~{srv-url}
         forwardAuthToken: true
         strictSSL: true
-    - name: bookshop-uaa
+    - name: bookshop-pg-uaa
     provides:
       - name: app-api
         properties:
           app-url: '${default-url}'
+
 # --------------------- RESOURCES ---------------------
 resources:
 # -----------------------------------------------------
-  - name: bookshop-uaa
+  - name: bookshop-pg-uaa
     type: org.cloudfoundry.managed-service
     parameters:
       service: xsuaa
       service-plan: application
       path: ./xs-security.json
       config: # override xsappname as it needs to be unique
-        xsappname: bookshop-${org}-${space}
+        xsappname: bookshop-pg-${org}-${space}
         oauth2-configuration:
           redirect-uris:
           - ~{app-api/app-url}/**
     requires:
       - name: app-api
-  - name: bookshop-hdi-container
+
+  - name: bookshop-pg-db
     type: org.cloudfoundry.managed-service
     parameters:
-      service: hana
-      service-plan: hdi-shared
+      path: ./postgresql-db-config.json
```

Comment:
Let's put the configuration directly in here under `config`, instead of separating this single line into a dedicated file.